package/scripts/setup-canvaskit.js

```diff
@@ -61,9 +88,48 @@ function copyFile(from, to) {
   fs.writeFileSync(to, data);
 }
 
-// Copy the WASM file to `<static>/static/js/canvaskit.wasm`
+function resolveFsAndPath() {
+  try {
+    const canvasKitPackageJsonPath = path.join(process.cwd(), 'node_modules', /* '@shopify', 'react-native-skia', 'node_modules', */ 'canvaskit-wasm', 'package.json');
+
+    console.log(
+      `› Optionnal metro bundler step:\n  ${gray(`Loading 'canvaskit-wasm' package.json from:\n  ${canvasKitPackageJsonPath}`)}`
+    );
+
+    const canvasKitPackageJson = require(canvasKitPackageJsonPath);
+    // eslint-disable-next-line import/no-dynamic-require
+    canvasKitPackageJson.browser = {
+      fs: false,
+      path: false,
+      os: false,
+    };
+    
+    console.log(`  ${gray(`Saving the updated 'canvaskit-wasm' package.json to resolve 'fs' and 'path' modules.`)}\n`);
+    
+    fs.writeFileSync(
+      canvasKitPackageJsonPath,
+      JSON.stringify(canvasKitPackageJson, null, 2)
+    );
+  } catch (error) {
+    // console.error(
+    //   `Could not update 'canvaskit-wasm' package.json to disable 'fs' and 'path' modules. Please install '@shopify/react-native-skia' and ensure it can be resolved from your project: ${process.cwd()}`
+    // );
+    console.log(error);
+    process.exit(1);
+  }
+}
+
 (() => {
-  copyFile(getWasmFilePath(), getOutputFilePath());
+  // Automatically detect if it's an expo project with a metro bundler
+  const isAnExpoProjectWithMetro = getWetherItsAnExpoProjectWithMetro();
+
+  // Copy the WASM file to `<static>/static/js/canvaskit.wasm`
+  copyFile(getWasmFilePath(), getOutputFilePath(isAnExpoProjectWithMetro));
+
+  // Resolve fs and path modules for metro bundler
+  if (isAnExpoProjectWithMetro === true) {
+    resolveFsAndPath();
```

Comment:
still not sure why we need this.

Reply:
I don't have any post install scripts, started from a blank expo-router template (SDK 50) and just copied the .wasm file manually